### ISSUE TITLE
[bitnami/aspnet-core] extra env vars

### ISSUE THREAD

**Issue by maxisam** [bitnami/charts#10803]
### Name and Version

bitnami/aspnet-core 3.4.6

### What steps will reproduce the bug?

it should be 

```
            {{- if .Values.extraEnvVars }}
            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
            {{- end }}
````
also ingress misses

```
  namespace: {{ include "common.names.namespace" . | quote }}
```

### What do you see instead?

https://github.com/bitnami/charts/blob/16569e01802bf3cd358da8a2d63669b0bb7bca31/bitnami/aspnet-core/templates/deployment.yaml#L136

**Comment by FraPazGal** [comment]
Hello @maxisam,

Thanks for reporting this, it looks like a typo. Would you be interested in contributing with a PR to fix it? We'll be glad to review and merge it.

**Comment by aoterolorenzo** [comment]
Hey @maxisam, thank you for taking the time to report the issue. I can inform you that it has been addressed today on #10857
